File: gb-spring-backend/src/main/java/com/egroupbuy/model/GroupBuyEvent.java
package com.egroupbuy.model;

import lombok.*;

import java.sql.Timestamp;


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Data
public class GroupBuyEvent {
    private int grpbuy_event_id;
    private Timestamp grpbuy_start;
    private Timestamp grpbuy_end;
    private String sell_type;
    private int product_id;
    private int package_grp_id;
    private int min_threshold;
    private int success_sold_qty;
    private String grpbuy_result;
    private Timestamp event_end_process;
    private String remark;
    private int seller_id;
}

File: gb-angular-frontend/src/app/interfaces/groupbuy-event.ts
export interface GroupBuyEvent {
  grpbuy_event_id: number;
  grpbuy_start: string;
  grpbuy_end: string;
  sell_type: string;
  product_id: number;
  package_grp_id: number;
  min_threshold: number;
  success_sold_qty: number;
  grpbuy_result: string;
  event_end_process: string;
  remark: string;
  seller_id: number;
}
